mozilla/foundation.mozilla.org
# file is now a .pdf instead of .txt

Closes #6414 


Link to test page: https://foundation-s-factory-pd-kv3dul.herokuapp.com/en/publication-page-with-child-article-pages/

**Steps to test**:

1. Please visit the test page above
2. Hover over the "Download PDF" Button, note that the file is now a PDF instead of a TXT like before


Docs Referenced: 
https://faker.readthedocs.io/en/master/providers/faker.providers.file.html#faker.providers.file.Provider
